config/config.md

```diff
@@ -263,122 +263,122 @@ As an alternative to `--db-config-file`, it allows specifying base64 encoded dat
 A LevelDB config file must be JSON and may have these keys.
 Any keys not given will receive the default value.
 
-```go
+```json
```

Comment:
This isn't valid json either (and the github visual of this block does _not_ like that it is marked as json)

Reply:
it's not a valid json because of the comments? either way this doc is completely non functional. Would rather please .md standards than GH because this doc is hosted on our dev docs

Reply:
Maybe have it as `text`? 🤔 Maybe json gets rendered as extended json depending on the rendering engine...

Reply:
text removes all color-coding which makes it a bit harder to read. 
<img width="782" alt="image" src="https://github.com/user-attachments/assets/4238d0bd-7be4-4c19-98fd-c89633e3fa61" />

Reply:
I readded the tab and set the codeblock back to go.

Reply:
https://github.com/DavidAnson/markdownlint/blob/main/doc/md010.md